index.html

```diff
@@ -100,132 +100,17 @@ <h2 class="projects-title">
       </h2>
       <div class="projects-container">
```

Comment:
Great job team 🤝

- [ ] Just like the previous reviewer pointed out, kindly ensure that your design fits properly on both mobile and desktop screens without using the scroll bar. For this case the vertical scroll.
- Your design
![your-design](https://user-images.githubusercontent.com/63917725/224029308-0724587f-406d-4808-a6af-10562cae9c09.PNG)
- Given design
![given-design](https://user-images.githubusercontent.com/63917725/224029376-266d8d3a-8083-441e-94f8-dab8f08b4334.PNG)

Reply:
@1GORDON I appreciate your help however, the scroll bar appears to allow the user to get a view of all information regarding a project. Some projects will have longer descriptions. 
Also, you added a comment on a section that is not responsible for the scrollbar could you be more specific in your comment and describe what scrollbar you would like me to take care of?

Reply:
The popup card is positioned as expected and includes an auto-scroll for when the content is more than what is shown in the design.